Src/VimWpf/Implementation/CommandMargin/CommandMarginController.cs

```diff
@@ -375,7 +375,7 @@ internal void HandleKeyEvent(KeyEventArgs e)
                     e.Handled = HandleHistoryNavigation(KeyInputUtil.VimKeyToKeyInput(VimKey.Up));
                     break;
                 case Key.Down:
-                    e.Handled = HandleHistoryNavigation(KeyInputUtil.VimKeyToKeyInput(VimKey.Up));
+                    e.Handled = HandleHistoryNavigation(KeyInputUtil.VimKeyToKeyInput(VimKey.Down));

```

Comment:
This look like it was accidental, but previous Down a synonym for Up.

Reply:
Yeah that definitely looks like a typo in the original code.